reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -54,13 +61,43 @@ class FreemarkerTemplateProcessor(
     companion object {
         const val OPTION_TEMPLATE_ID = "template.id"
         const val OPTION_TEMPLATE_PATH = "template.path"
+        const val OPTION_PROJECT_TYPES_AS_PACKAGES = "project.types.as.packages"
```

Comment:
I completely missed that property name before, dots should be used to create a hierarchy, like with "template.id" and "template.path". This should be called "project-types-as-packages" instead. Also, this new option must be added to the class docs of all reports that use the `FreemarkerTemplateProcessor`.

Reply:
Done.